src/python/pants_release/generate_release_announcement.py

```diff
@@ -47,7 +47,7 @@ def announcement_text() -> str:
                 announcement += "\n"
             announcement += (
                 "\nWelcome to the Pants community! We appreciate your contributions, "
-                "and look forward to more."
+                "and look forward to more of them in the future."
```

Comment:
This is more grammatical, and looks better in Slack, where otherwise the word "more" ends up on the last line on its own.